CHANGELOG.md

```diff
@@ -1,4 +1,96 @@
+14. Implemented support for updating certificate domain:
+  - New resource **aviatrix_controller_cert_domain_config**
+15. Implemented support for TGW segmentation for Egress:
+  - New attribute ``tgw_segmentation_for_egress_enabled`` in **aviatrix_firenet** and data source
+16. Implemented support for OCI FireNet
+17. Implemented support for Egress FireNet route injection:
+  - New attribute ``egress_static_cidrs`` in **aviatrix_firenet** and data source
+18. Implemented support for launching user provided AMI for FireNet:
+  - New attribute ``firewall_image_id`` in **aviatrix_firewall_instance**
+19. Implemented support for optional tunnel ip addresses with following attributes in **aviatrix_site2cloud**:
+  - ``local_tunnel_ip``
+  - ``remote_tunnel_ip``
+  - ``backup_local_tunnel_ip``
+  - ``backup_remote_tunnel_ip``
+20. Implemented support for phase 1 remote identifier for Site2Cloud:
+  - New attribute ``phase1_remote_identifier`` in **aviatrix_site2cloud**
+21. Implemented support for AzureGov, AWSChina and AzureChina clouds in resources **aviatrix_account**, **aviatrix_vpc**, **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+22. Implemented support for creating an **aviatrix_vpc** in Azure with an existing resource group ``resource_group``
+23. Implemented support for separate IAM role and policy for gateways in AWS **aviatrix_account** with new attributes:
+  - ``aws_gateway_role_app``
+  - ``aws_gateway_role_ec2``
+24. Implemented support for IPSec tunnel down detection time in **aviatrix_gatewway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**:
+  - New attribute ``tunnel_detection_time``
+25. Implemented support for **aviatrix_site2cloud** connections to have a single public IP with two gateways:
+  - New attribute ``single_ip_ha``
+26. Implemented support for enabling auditing in **aviatrix_account**:
+  - New attribute ``audit_account``
+27. Implemented support for BGP max AS limit controller configuration:
+  - New resource **aviatrix_controller_bgp_max_as_limit_config**
+28. Implemented support for OCI insane mode in **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+29. Implemented support for ``enable_egress_transit_firenet`` for Azure and OCI in **aviatrix_transit_gateway**
+
+### Enhancements:
+1. Added following attributes in **aviatrix_account** data source:
+  - ``gcloud_project_id``
+  - ``arm_subscription_id``
+  - ``awsgov_account_number``
+  - ``awsgov_access_key``
+2. Optimized state refresh performance for **aviatrix_gateway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
+3. Added support for Fortinet Fortigate in **data_source_aviatrix_firenet_vendor_integration**
+4. Added computed value ``tgw_id`` in **aviatrix_aws_tgw**
+5. Added new map type attribute ``tags`` to replace ``tag_list`` in **aviatrix_gatewway**, **aviatrix_spoke_gateway** and **aviatrix_transit_gateway**
```

Comment:
typo in **aviatrix_gateway**

- also, is it ok to have this point under enhancements, as this is also a duplicate as a deprecation point # 2 ?